Sources/Conformance/main.swift

```diff
@@ -120,6 +120,8 @@ func singleTest() throws -> Bool {
 }
 
 Google_Protobuf_Any.register(messageType: Conformance_TestAllTypes.self)
+Google_Protobuf_Any.register(messageType: ProtobufTestMessages_Proto3_TestAllTypes.self)
```

Comment:
I haven't looked - What's the threading on this?  i.e. - Since it is a global registry what happen if some some subsystem is registering while another us using an Any?

For [ObjC I avoid the registry](https://github.com/google/protobuf/blob/master/objectivec/GPBWellKnownTypes.h#L216-L230), seems like we might be able to template the function to take a Swift protobuf class instead and confirm the Any is that type before extracting instead of relying on a registry?